.eslintrc.js

```diff
@@ -17,25 +17,29 @@ module.exports = {
   rules: {
     'no-console': 'error',
     'prettier/prettier': [
-      'error', {
-        'endOfLine': 'auto'
-      }
+      'error',
+      {
+        endOfLine: 'auto',
+      },
     ],
-    // TODO remove when done killing `any`s and making tsc strict
-    '@typescript-eslint/ban-ts-comment': 'off',
-    '@typescript-eslint/no-explicit-any': 'off',
-    '@typescript-eslint/no-unsafe-assignment': 'off',
-    '@typescript-eslint/no-unsafe-call': 'off',
-    '@typescript-eslint/no-unsafe-member-access': 'off',
-    '@typescript-eslint/no-unsafe-return': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'warn',
+    '@typescript-eslint/no-confusing-non-null-assertion': 'warn',
+    '@typescript-eslint/no-explicit-any': 'warn',
+    '@typescript-eslint/no-extraneous-class': 'warn',
+    '@typescript-eslint/no-implicit-any-catch': 'warn',
+    '@typescript-eslint/no-invalid-void-type': 'warn',
+    '@typescript-eslint/prefer-ts-expect-error': 'warn',
+    '@typescript-eslint/type-annotation-spacing': 'warn',
+    '@typescript-eslint/typedef': 'warn',
+    '@typescript-eslint/unified-signatures': 'warn',
```

Comment:
From a quick look, I like the list of rules you picked. One question, though: how did you pick this exact list of rules? 

Why them, why not less, why not more from https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#supported-rules ?

Reply:
I skipped ones that were supported in the preset, and from there just used my best judgement on what seemed strict from a typing/any perspective. I'm flexible on adding/removing these.